src/cogs/github_cog.py

```
@@ -180,6 +181,291 @@ async def github_search(self, interaction: discord.Interaction, query: str):
+        app_commands.Choice(name="This Month", value="this_month")
+    ])
+    @app_commands.choices(type=[
+        app_commands.Choice(name="Repositories", value="repositories"),
+        app_commands.Choice(name="Developers", value="developers")
+    ])
+    async def github_trending(
+        self,
+        interaction: discord.Interaction,
+        date_range: str,
+        type: str,
+        language: str = None
```

Ruff (0.14.2)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)